spec/core_functions/meta/module_mixins.hrx

```diff
@@ -0,0 +1,372 @@
+<===> options.yml
+---
+:todo:
+- sass/libsass#2807
+
+<===> _util.scss
+@use "sass:meta";
+
+@mixin print-mixin-map($mixins) {
+  a {
+    @each $name, $mixin in $mixins {
+      #{$name}: {@include meta.apply($mixin)};
+    }
+  }
+}
+
+<===>
+================================================================================
+<===> empty/input.scss
+@use "sass:meta";
+@use "other";
+
+a {b: meta.inspect(meta.module-mixins("other"))}
+
+<===> empty/_other.scss
+// This module defines no mixins.
+
+<===> empty/output.css
+a {
+  b: ();
+}
+
+<===>
+================================================================================
+<===> multiple/input.scss
+@use "sass:meta";
+@use "../util";
```

Comment:
It's more reliable to refer to this as `core_functions/meta/utils`, since then you can freely change the path of the test case without updating this relative path.